### PR TITLE
Adding documentation for monitoring and cluster logging.

### PR DESCRIPTION
**Pull request by computate**

### CODE REVIEW

File: logging.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- The OpenShift Logging Operator has a dependency on the [Elasticsearch Operator here](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/openshift-operators-redhat/operators.coreos.com~v1alpha1~Subscription/elasticsearch-operator). Whether you use Elasticsearch for storing logs or using Loki, you still need the Elasticsearch Operator installed for required dependent CustomResourceDefinitions.
+
+## Loki operator
+
+- The Loki Operator allows you to setup LokiStacks, AlertingRules, RecordingRules, and RulerConfigs based on your cluster logs for infrastructure, audit, and applications. See the [ Loki Operator here ](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/openshift-operators-redhat/operators.coreos.com~v1alpha1~ClusterServiceVersion/loki-operator.5.4.6)
```
Comment (joachimweyl) [inline logging.md]: `setup` -> `set up`?

File: logging.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- Setting up a LokiStack involves configuring an object storage by a secret named "thanos-object-storage" in the "openshift-operators-redhat" namespace containing the access_key_id, access_key_secret, bucketnames, and endpoint of the object storage.
+- The object storage for Loki is provided by OpenShift Data Foundations. See the [ openshift-operators-redhat-objectbucketclaim Object Bucket Claim here ](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/openshift-operators-redhat/objectbucket.io~v1alpha1~ObjectBucketClaim/openshift-operators-redhat-objectbucketclaim)
+
+## Grafana operator
+
+- The Grafana Operator allows you to setup Grafana instances, datasources, dashboards, and notification channels. See the [ Grafana Operator here ](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/grafana/operators.coreos.com~v1alpha1~ClusterServiceVersion/grafana-operator.v4.6.0)
```
Comment (joachimweyl) [inline logging.md]: `setup` -> `set up`?

File: logging.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- Setting up a LokiStack allows you to configure the size of a cluster logging system that you desire in terms of storage and replicas. [ LokiStack here ](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/openshift-operators-redhat/loki.grafana.com~v1beta1~LokiStack/lokistack)
+- Setting up a LokiStack involves configuring persistent storage by storageClassName for Persistent Volume Claims. [ ocs-external-storagecluster-ceph-rbd storage class here ](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/cluster/storageclasses/ocs-external-storagecluster-ceph-rbd)
+- Setting up a LokiStack involves configuring an object storage by a secret named "thanos-object-storage" in the "openshift-operators-redhat" namespace containing the access_key_id, access_key_secret, bucketnames, and endpoint of the object storage.
```
Comment (joachimweyl) [inline logging.md]: `an object storage` -> `object storage`?

File: logging.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+
+# Tracking events in the Logging System
+
+As an administrator of the cluster, I should be able to track all the events in the cluster using the logging system in OpenShift.
+
+## Steps
+
+- [Click here to visit the Logging Grafana instance](https://logging-grafana.apps.nerc-ocp-infra.rc.fas.harvard.edu).
+- Click on the "Explore" button which looks like a compass on the left side. At the top of that page, you can select from the dropdown the infrastructure, audit or application logs. You can then use the Log Browser to browse the different labels like kubernetes_host and select a value for a tag to filter in on actual search results. Continue to add more labels and values to narrow in on the logs you wish to see.
```
Comment (joachimweyl) [inline logging.md]: `audit or` -> `audit, or`?

File: monitoring.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+## Monitoring and logging for the infrastructure hardware and software that is not OpenShift (for example Grafana).
+
+As a NERC administrator, I should be able to monitor the status of any infrastructure software or hardware that supports operations for the NERC OpenShift environment, even if it is not itself part of OpenShift.
+
+## Steps for reporting
+
+You can access many metrics for pods of a particular application like grafana or loki.
+
+- [Click here to visit the cpu usage logs for grafana](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate%7Bnamespace%3D%5C%22grafana%5C%22%7D%22%7D%5D).
```
Comment (joachimweyl) [inline monitoring.md]: `cpu` -> `CPU`?

File: monitoring.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+- [Click here to visit the cpu usage logs for loki](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate%7Bnamespace%3D%5C%22openshift-operators-redhat%5C%22%7D%22%7D%5D).
```
Comment (joachimweyl) [inline monitoring.md]: `cpu` -> `CPU`?

File: monitoring.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+# Using the Reporting tools
+
+## Track/report usage of the cluster
+
+As an administrator of the cluster, I should be able to view daily, weekly and monthly reports of the cluster infrastructure utilization.
```
Comment (joachimweyl) [inline monitoring.md]: `weekly and` -> `weekly, and`

File: monitoring.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+### Steps
+
+- Administrator logs into the associated XDMoD instance and views reports.
+- [Click here to view the ACM Observability Grafana dashboards](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/). These dashboards provide insights into Control Plane Health, Optimization, Capacity, Utilization and more. You can change the timespan in the top right to show results in terms of minutes, hours, days, months or years.
+
+## Track/report usage of the project
+As a user and the owner of a project, I should be able to view daily, weekly and monthly reports of the infrastructure utilization by the projects I own.
```
Comment (joachimweyl) [inline monitoring.md]: `weekly and` -> `weekly, and`

File: monitoring.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+# Using the Reporting tools
+
+## Track/report usage of the cluster
+
+As an administrator of the cluster, I should be able to view daily, weekly and monthly reports of the cluster infrastructure utilization.
+
+### Steps
+
+- Administrator logs into the associated XDMoD instance and views reports.
+- [Click here to view the ACM Observability Grafana dashboards](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/). These dashboards provide insights into Control Plane Health, Optimization, Capacity, Utilization and more. You can change the timespan in the top right to show results in terms of minutes, hours, days, months or years.
```
Comment (joachimweyl) [inline monitoring.md]: `Utilization and` -> `Utilization, and`

File: monitoring.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+## Track/report usage of the project
+As a user and the owner of a project, I should be able to view daily, weekly and monthly reports of the infrastructure utilization by the projects I own.
+
+### Steps to track usage
+
+- User logs into the associated XDMoD instance and views reports for projects they own.
+- User cannot view reports for projects they do not own. We will need to look into this, to restrict the view to only projects that they own.
```
Comment (joachimweyl) [inline monitoring.md]: `User` -> `Users`

File: monitoring.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+- [Click here to view the memory usage of projects over time](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22namespace:container_memory_usage_bytes:sum%22%7D%5D).
+- [Click here to view the cpu usage of the projects over time](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22node_namespace_pod_container:container_cpu_usage_seconds_total:sum%22%7D%5D).
```
Comment (joachimweyl) [inline monitoring.md]: `cpu` -> `CPU`

File: monitoring.md [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+- [Click here to show the projects using the top 5 cpu usage at each point in time](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22topk%285,%20%281%20-%20avg%28rate%28node_namespace_pod_container:container_cpu_usage_seconds_total:sum%7B%7D%5B$__rate_interval%5D%29%29%20by%20%28namespace%29%29%29%22%7D%5D).
```
Comment (joachimweyl) [inline monitoring.md]: `cpu` -> `CPU`

**Comment by joachimweyl** [comment]
I found a few nitpicky grammatical things but the details and instructions look great.

**Comment by computate** [comment]
Thanks for the review @joachimweyl , I'll review my grammar next time :).

**Review by joachimweyl — APPROVED** [review]

**Pull request merged**